### ISSUE TITLE
Pre-set state for in-game briefing

### ISSUE THREAD

**Issue by TheLocalPub** [Global-Conflicts-ArmA/Olsen-Framework-Arma-3#220]
Is there a possibility to set a side to ready before mission start.
Mission will go live only when one team readies up.

**Comment by PiZZAD0X** [comment]
simply set `IGBRIEF_bluforIsReady` or `IGBRIEF_opforIsReady` to true in the postInitGlobal customization file

**Comment by TheLocalPub** [comment]
So to clarify. 
There no state for INDFOR?

**Comment by PiZZAD0X** [comment]
no, only blufor/opfor. The sides are not directly used, you could use it for opfor vs indfor, the boards and their placement are the important thing

**Comment by EM-Creations** [comment]
Is this still an issue? What's the ask, to make an IGBRIEF_indforIsReady and IGBRIEF_civforIsReady variable?

**Comment by TheLocalPub** [comment]
I was originally asking to allow the IGB module to allow a mission maker to set a side to "ready" at mission start.
An example of this is let's say you have an attack/defend mission, I once had a missions like this where I wanted the defending team to choose when they wanted to start the attack from blufor so to speak.
The ready-up period would be the time they have to plan, brief, and then go out and set up defences.
Once the defending team is happy with their defences, they personally can start the mission/make it go live.

But to also allow INDFOR to be a side that can use this.
I don't see why it just doesn't have the ability to use blufor, opfor, indfor, and civfor, for all intents and purposes.
